react-toggle-switch-component/my-toggle-button/src/ToggleSwitch.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import './ToggleSwitch.css';
+
+export default function ToggleSwitch({ label }) {
+  const [toggled, setToggle] = useState(true);
+
+  if (toggled) {
```

Comment:
You are overriding the `label` prop here, so you should make it a local variable instead of a prop.